sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/api/ContactApiService.java

```diff
@@ -18,24 +18,28 @@
 
 package org.sormas.e2etests.services.api;
 
-import com.google.inject.Inject;
 import java.util.Date;
 import java.util.UUID;
 import org.sormas.e2etests.pojo.api.*;
 
 public class ContactApiService {
 
-  @Inject
-  public ContactApiService() {}
-
   public Contact buildGeneratedContact(Person person) {
     return Contact.builder()
         .disease("CORONAVIRUS")
         .uuid(UUID.randomUUID().toString())
         .reportDateTime(new Date())
         .reportingUser(ReportingUser.builder().uuid("QLW4AN-TGWLRA-3UQVEM-WCDFCIVM").build())
-        .district(District.builder().uuid("SZ75BK-5OUMFU-V2DTKG-5BYACHFE").build())
-        .region(Region.builder().uuid("SZ75BK-5OUMFU-V2DTKG-5BYACHFE").build())
+        .district(
+            District.builder()
+                .caption("Voreingestellter Landkreis")
+                .uuid("SZ75BK-5OUMFU-V2DTKG-5BYACHFE")
+                .build())
+        .region(
+            Region.builder()
+                .caption("Voreingestellte Bundesl\u00E4nder")
+                .uuid("RKVAOM-ZNAAFU-R2KF6Z-6BENKHEY")
```

Comment:
agree but its not the first time, a lot of stuff is hardcoded already it will come back to bite us if the test environment is reset or changed, however I don't see any other option atm (maybe collecting, reading all of these uuids of users, regions, districts at the beginning of the testcases ?) Tamas what is your opinion ?

Reply:
We need to implement the the get regions and populate everything from there.